wagtail/admin/templates/wagtailadmin/pages/side_panels/preview.html

```diff
@@ -1 +1,42 @@
-{# TODO: Page history panel #}
+{% load i18n wagtailadmin_tags %}
+
+{% auto_update_preview as auto_update %}
```

Comment:
I’m surprised we’d want to follow this setting for the preview panel – I don’t think I’ve ever seen a side-by-side preview panel that isn’t live-updating.

Reply:
Maybe it's worth to keep, but make it default to `True`? (I just added this change.)